### PR TITLE
fix evm test inputs

### PR DESCRIPTION
**Pull request by leonardoalt**
Fixing the EVM test inputs and trying it out on our CI again. If CI fails because of OOM I'll comment the function out but will leave the updated inputs and `verify_riscv_crate` adjustment.

### CODE REVIEW

File: riscv/tests/riscv.rs [hunk 1]
```diff
@@ -124,21 +124,21 @@ fn test_password() {
 mstore(0, 666)
 return(0, 32)
 */
-/*
 static BYTECODE: &str = "61029a60005260206000f3";
```
Comment (chriseth) [inline riscv/tests/riscv.rs:127]: This could use PUSH0 ;)

**Review by chriseth — DISMISSED** [review]

**Comment by leonardoalt** [comment]
updated this PR to run the evm test with the slow tests, let's see

**Comment by leonardoalt** [comment]
yea this is still too much for our CI...

**Comment by leonardoalt** [comment]
I commented out the test again because of CI, but at least the test is correct now if you uncomment and run it

**Comment by leonardoalt** [comment]
the test is at least compiled now, hidden behind a feature.

Ready for review

**Comment by chriseth** [comment]
looks good, but tests are failing

**Review by lvella — APPROVED** [review]

**Pull request merged**